code/modules/clothing/chameleon.dm

```diff
@@ -420,9 +429,87 @@
 	chameleon_action.chameleon_blacklist = typecacheof(/obj/item/weapon/gun/magic, ignore_root_path = FALSE)
 	chameleon_action.initialize_disguises()
 
+/obj/item/weapon/gun/energy/laser/chameleon/Initialize()
+	..()
+	chameleon_projectile_vars = list("name" = "practice laser", "icon" = 'icons/obj/projectiles.dmi', "icon_state" = "laser")
+	copy_vars = list("name", "icon", "icon_state", "speed", "color", "hitsound", "forcedodge", "impact_effect_type", "range", "suppressed", "hitsound_wall", "impact_effect_type", "pass_flags")
```

Comment:
Shouldn't really copy `forcedodge` or `pass_flags` as that'll let your fake projectile start i.e. passing through mobs; `speed` and `range` too but they probably won't matter much.